### PR TITLE
Trying a new way to copy themes. Everything BUT defaults. TEST THIS

### PR DESCRIPTION
**Pull request by shanselman**
Please read this carefully. We're using wildcards now. Config is copied, Default Themes are PACKED, except the themes YOU add.

### CODE REVIEW

File: source/DasBlog.Web.UI/DasBlog.Web.csproj [hunk 1]
```diff
@@ -99,24 +83,18 @@
...
-    <None Update="Themes\dasblog\custom.css">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="Themes\fulcrum\custom.css">
+    <None Include="Config\*">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
-    <None Update="Themes\median\custom.css">
+    <!-- Just copy themes that ARE NOT defaults and everything else is in a compiled razor pack -->
+    <None Include="Themes\**" Exclude="Themes\DasBlog\*;Themes\fulcrum\*;Themes\median\*;Themes\_ViewImports.cshtml;Themes\_ViewStart.cshtml">
```
Comment (shanselman) [inline source/DasBlog.Web.UI/DasBlog.Web.csproj]: This is the magic.

File: source/DasBlog.Web.UI/Properties/launchSettings.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "http://localhost:50432/"
+      "applicationUrl": "https://localhost:5001;http://localhost:5000"
```
Comment (shanselman) [inline source/DasBlog.Web.UI/Properties/launchSettings.json]: We are also switching to Kestrel for local dev which is more reliable than IIS Express

File: source/DasBlog.Web.UI/DasBlog.Web.csproj [hunk 3]
```diff
@@ -99,24 +83,18 @@
...
   </ItemGroup>
   <ItemGroup>
-    <None Update="Config\IISUrlRewrite.xml">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="Themes\dasblog\custom.css">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="Themes\fulcrum\custom.css">
+    <None Include="Config\*">
```
Comment (shanselman) [inline source/DasBlog.Web.UI/DasBlog.Web.csproj]: all config files are copied from here.

**Comment by poppastring** [comment]
@shanselman I could not get this PR working last night, I will take another look this evening.

**Comment by shanselman** [comment]
Cool, I'm around Friday if you want to pair together and look.
--
Scott Hanselman
@shanselman <http://twitter.com/shanselman> - Podcast:
https://hanselminutes.com/shows

**Pull request merged**